src/main/java/org/folio/services/migration/async/PublicationPeriodMigrationService.java

```diff
@@ -12,14 +12,12 @@
 import org.folio.rest.persist.PgUtil;
 import org.folio.rest.persist.PostgresClientFuturized;
 import org.folio.rest.persist.SQLConnection;
-import org.folio.services.instance.InstanceEffectiveValuesService;
 
 public class PublicationPeriodMigrationService extends AsyncBaseMigrationService {
```

Comment:
Let's remove this class and everything related to publication period migration also.

Reply:
done